browser/css/jssidebar.css

```diff
@@ -329,3 +329,12 @@ button#button2.ui-pushbutton.jsdialog.sidebar {
 .sidebar .spinfieldcontainer input {
 	width: calc(100% - 28px);
 }
+
+#NumberFormatPropertyPanel > tr:nth-child(1) > td:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2),
+#NumberFormatPropertyPanel > tr:nth-child(1) > td:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) {
```

Comment:
This works but to have this type of nth-child chain is not that good, in terms of how it looks/maintainability and more important because probably in short time this change will stop to work due to any small change on the sidebar structure be that from online or core